docs/Public_API.rst

```diff
@@ -243,7 +243,7 @@ AutoScaler collects the instances' metrics of an application, and aggregate the
 Route
 '''''
 
-    GET /v1/apps/8d0cee08-23ad-4813-a779-ad8118ea0b91/aggregated_metric_histories/memoryused
+    GET /v1/apps/:guid/aggregated_metric_histories/memoryused
 
 Parameters
```

Comment:
For the **Parameters** definition:
1. metric-type:
    - `metric_type` with '_' instead of '-' is used in `GET /v1/apps/:guid/metric_histories/:metric_type`
    - need we add custom metric in the description?
2. order-direction: `The scaling history will be order by timestamp asc or desc.` -> `The metric history will be order by timestamp asc or desc.`